src/checkers/inference/solver/MaxSat2TypeSolver.java

```diff
@@ -113,7 +111,9 @@ public InferenceSolution solve() {
 
                     Integer potential = existentialToPotentialIds.get(var);
                     if (potential != null) {
-                        idToExistence.put(potential, !isTop);
+                        // idToExistence.put(potential, !isTop);
+                        // TODO: which AnnotationMirror should be used?
```

Comment:
Because idToExistence is not clearly defined. What does it mean to add `!isTop` as value for potential to idToExistence, but not add anything to the result? Is it guaranteed that someone else will add something to result? What does it mean if idToExistence gives true, but result doesn't contain anything?
I didn't find a clear description of what the intended semantics is and don't know whether making potential be bottom is the same thing.